docs/getting-started/overview.md

```diff
@@ -1,4 +1,4 @@
-# Getting Started
+# Overview
```

Comment:
Since getting start has only one page, can we use this page directly?

Reply:
I'm not sure, maybe @alili has great ideas